code/modules/mob/living/silicon/pai/pai_defense.dm

```diff
@@ -60,10 +60,11 @@
 	return FALSE //No we're not flammable
 
 /mob/living/silicon/pai/proc/take_holo_damage(amount)
-	emitterhealth = CLAMP((emitterhealth - amount), -50, emittermaxhealth)
-	if(emitterhealth < 0)
-		fold_in(force = TRUE)
-	to_chat(src, "<span class='userdanger'>The impact degrades your holochassis!</span>")
+	if(amount > 0)
+		emitterhealth = CLAMP((emitterhealth - amount), -50, emittermaxhealth)
+		if(emitterhealth < 0)
+			fold_in(force = TRUE)
+		to_chat(src, "<span class='userdanger'>The impact degrades your holochassis!</span>")
```

Comment:
Maybe the `if(amount > 0)` check should be moved down to the chat message and negative amounts let heal the pAI, don't you think?